### PR TITLE
[community] Continued updates to Projects design / layout

### PR DESCRIPTION
**Pull request by moonclavedev**
### Description
Update Projects design

**Projects Landing:**

- Update hero to full width
- New hero image & headline copy
- Insert Search Field (needs to be connected)
- Insert Filter by category field (needs to be connected)
- `position:sticky` container for Search, Filter & Submit Project 
- Simplify project cards by removing full description, adjust styles
- Page titles & Meta descriptions

**Projects Submit:**
- Updated form styles
- `position:sticky` subheadings
- Page titles & Meta descriptions

**Project Single:**
- Show project thumbnail on mobile
- Setup CTA button for visiting project website as opposed to text link
- Create "back to Projects button" (needs further iteration, such as `href="javascript:window.history.go(-2)"` if referring page is `/projects` to retain scroll position
- `position:sticky` sidebar

### Test Plan
IMG_3114

image

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4223)

### CODE REVIEW

File: ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css [hunk 1]
```diff
@@ -11,11 +11,6 @@
   }
 }
 
-html {
-  /* So the sticky header doesn't cover things that are scrolled into view. */
-  scroll-padding-top: 120px;
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css:16]: Why is this removed? Without it, say you go to an #anchor link, isn't it covered by the HeaderComponent?

**Review by zacharydenton — APPROVED** [review]

File: ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css (hunk 1 above)
Comment (moonclavedev) [reply ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css:16]: It's still there, just moved to a utility class: 

https://tailwindcss.com/docs/scroll-padding

https://github.com/aptos-labs/aptos-core/pull/4223/commits/e77ab593aee20a72616036c443a95f9d03aa24c2#diff-cee8494b29ca08559a959775a9bd5dce3345c07d377a7299a8b62b9284c477ad

**Review by davidiw — APPROVED** [review]

File: ecosystem/platform/server/app/views/projects/show.html.erb [hunk 2]
```diff
@@ -52,5 +59,6 @@
         <% end %>
       </div>
     </div>
+
```
Comment (davidiw) [inline ecosystem/platform/server/app/views/projects/show.html.erb:62]: oh why!
